packages/keystone/src/admin-ui/system/createAdminMeta.ts

```diff
@@ -98,7 +98,6 @@ export function createAdminMeta(
     }
 
     for (const [fieldKey, field] of Object.entries(list.fields)) {
-      if (field.access.read === false) continue;
```

Comment:
Could you add this back(but based on isEnabled) until we deal with this properly then?